src/main/webapp/homepage_style.css

```diff
@@ -313,3 +313,16 @@ section {
     font-size: 2.5em;
   }
 }
+
+#task-detail-container {
+  margin-bottom: 20px;
+}
+
+.task-content {
+  cursor: pointer;
+}
+
+.task-content:hover {
```

Comment:
To keep the page consistent, I'd rather you didn't use this styling but instead the same styling used by a:hover so it's consistent with the rest of the homepage.

Reply:
Sure. I will change it in the new commit